config/config.php

```diff
@@ -12,24 +12,15 @@
  * Obviously, if you use closures in your config you can't cache it.
  */
 
-$cachedConfigFile = 'data/cache/app_config.php';
+$configManager = new ConfigManager([
+    // Load config from all modules first
+    new PhpFileProvider('src/*/Config/{{,*.}global,{,*.}local}.php'),
```

Comment:
> Is it correct that you want to place config files in the /src/ dir of each module?

Not sure what you mean. The idea behind this config manager is to make it easier to create modules and load their config. So yes, the config files are placed inside the modules. If you mean they are placed at the same level as ModuleName/Action etc, then yes, since there is no ModuleName/src/Action etc. But that's for @weierophinney to decide if we change the structure. I wanted to keep it as close as possible to your tutorial.

> And that it is possible to have global.php and local.php within the modules?

I agree, local.php should be in config/autoload/.

But what is the best to do? Only allow global.php or just .php?